docs/tutorial/batch_predict/batch_predict.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
```

Comment:
<p>Project name should be different</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/2415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Recommend taking this up with haviv :)</p>